plutus-core/test/Spec.hs

```diff
@@ -46,8 +45,7 @@ main = do
     rwFiles <- findByExtension [".plc"] "test/scopes"
     typeFiles <- findByExtension [".plc"] "test/types"
     typeErrorFiles <- findByExtension [".plc"] "test/type-errors"
-    evalFiles <- findByExtension [".plc"] "test/Evaluation/Golden"
```

Comment:
Were we running golden tests twice: once here (only for the CEK machine) and once in `Golden.hs` (for both the CK and CEK machines)?

Reply:
Probably. I've been meaning to tidy up this code for ages...